src/karmen_frontend/src/routes/printers/printer-detail.js

```diff
@@ -166,17 +166,33 @@ const PrinterDetail = ({
                     Disconnect
                   </button>
                 ))}
+              {printer.client.pill_info ? (
+                printer.client.pill_info.update_available ? (
+                  <div className="printer-state-announcement">
+                    There is update available for your pill{" "}
```

Comment:
Update available for your Pill

Reply:
An update is available for your Pill

Reply:
247da114d0cb0121f6a38543ab3f6ec7bd540bf1